setup.py

```diff
@@ -62,7 +62,7 @@ def set_sysv_files(data_files, dest="/etc/rc.d/init.d", src=None):
     data_files.append((dest, src))
 
 
-def set_systemd_files(data_files, dest="/lib/systemd/system", src=None):
+def set_systemd_files(data_files, dest, src=None):
```

Comment:
Can you remove the 2 slice files on line 70? #2145 moves them to the correct place, but our PRs will be merged to different releases

Reply:
Would it throw an error if we let it be for this release? Like I would prefer if it be bundled with your main PR but if its susceptible to errors then I can remove it now too

Reply:
we would create unit files that are not used by anybody (and they would be in a system directory)

Reply:
sounds good, let me remove them then